drivers/hmis/app/models/hmis/role.rb

```diff
@@ -397,6 +397,13 @@ def self.permissions_with_descriptions
         category: 'Administration',
         sub_category: 'Admin Tools',
       },
+      can_administrate_config: {
+        description: 'Ability to use the JSON form editor, full Form Builder, and fragment management.',
```

Comment:
```suggestion
        description: 'Ability to use the JSON form editor and administer all form types [GR Staff Only]',
```
?
(don't want it to sound too enticing 😄 )